core/common/src/main/java/alluxio/underfs/UnderFileSystemRegistry.java

```diff
@@ -111,7 +111,8 @@ public static UnderFileSystem create(String path, Map<String, String> ufsConf) {
     for (UnderFileSystemFactory factory : factories) {
       try {
         // Use the factory to create the actual client for the Under File System
-        return new UnderFileSystemWithLogging(factory.create(path, ufsConf));
+        return new UnderFileSystemWithLogging(
+            factory.create(path, new UnderFileSystemConfiguration(ufsConf)));
```

Comment:
the line right now is in `UnderFileSystemRegistry.java`. You can think in this way that anything in `underfs` module is referencing `UnderFileSystemConfiguration` (and that's why it is in `alluxio.underfs`), whereas above this class, using Map as it is what passed around in RPC